util/src/main/java/com/ibm/wala/util/PlatformUtil.java

```diff
@@ -47,38 +46,44 @@ public static boolean onIKVM() {
   }
 
   /**
-   * get the jars in the boot classpath. TODO test on more JVMs
+   * Gets the standard JDK modules shipped with the running JDK
    *
-   * @throws IllegalStateException if boot classpath cannot be found
+   * @param justBase if {@code true}, only include the file corresponding to the {@code java.base}
+   *     module
+   * @return array of {@code .jmod} module files
+   * @throws IllegalStateException if modules cannot be found
    */
-  public static String[] getBootClassPathJars() {
-    String classpath = null;
-    String javaVersion = System.getProperty("java.specification.version");
-    if (!javaVersion.equals("1.8")) {
-      // java11 support for jmod files
+  public static String[] getJDKModules(boolean justBase) {
```

Comment:
I'm all about that bass, 'bout that bass, no treble
I'm all about that bass, 'bout that bass, no treble
I'm all about that bass, 'bout that bass (bass, bass, bass, bass)

🎶